src/genLink.ts

```diff
@@ -39,17 +45,70 @@ const genLineNumber = (select: vscode.Selection, useSubPrefix: boolean = false):
 
   startLine = startLine + 1;
   if (endChar !== 0) endLine = endLine + 1;
-  if (startLine === endLine) return `#L${startLine}`;
-  
-  return `#L${startLine}-${useSubPrefix ? 'L' : ''}${endLine}`;
+  if (startLine === endLine) {
+    // Only highlight 1 line
+    switch (gitHost) {
+      case GitHost.bitbucket:
+        return `#lines-${startLine}`;
+
+      default:
+        return `#L${startLine}`;
+    }
+  } else {
+    // Highlight multiple lines
+    switch (gitHost) {
+      case GitHost.bitbucket:
+        return `#lines-${startLine}:${endLine}`;
+
+      default:
+        return `#L${startLine}-L${endLine}`;
```

Comment:
Gitlab use `FILE_NAME#L1-20` way to show multiple lines, so you should add another `case GitHost.Gitlab` before `default`.

Reply:
I was also able to get Gitlab to work with a secondary `L`, [like so](https://gitlab.com/bramw/baserow/-/blob/develop/docker-compose.local.yml#L4-L28). It might be that [originally they didn't have it](https://gitlab.com/gitlab-org/gitlab/-/issues/7937) and added it later to bring it to parity with Github? Either way, do you still want me to remove the L for Gitlab?

Reply:
Oh I didn't notice this behaviour before, it looks like a hidden feature in Gitlab. For that you don't need to modify it :)